Misc/NEWS.d/next/Core and Builtins/2018-03-13-14-46-03.bpo-32949.v821M7.rst

```diff
@@ -0,0 +1,5 @@
+Removed WITH_CLEANUP_START, WITH_CLEANUP_FINISH, BEGIN_FINALLY, END_FINALLY, CALL_FINALLY and POP_FINALLY bytecodes.
+Replaced with RERAISE and WITH_EXCEPT_FINISH bytecodes.
+The compiler now generates different code for exceptional and non-exceptional branches for 'with' and 'try-except'
+statements. For 'try-finally' statements the 'finally' block is replicated for each exit from the 'try' body.
```

Comment:
I hope this is misleading.  If the try block has 10 lines, and one of those lines contains 3 function calls ... does that mean there will be a dozen physical copies of the finally block?

Reply:
I don't see the relevance of the number of function calls in the try body. So the answer to your question is probably no.
There is one bytecode version of the finally block per exit (fall-through, return or exception) from the try body.